Tests/AppTests/SearchTests.swift

```diff
@@ -36,46 +38,58 @@ class SearchTests: AppTestCase {
 
     func test_packageMatchQuery_single_term() throws {
         let b = Search.packageMatchQueryBuilder(on: app.db, terms: ["a"], filters: [])
-        XCTAssertEqual(renderSQL(b), #"SELECT 'package' AS "match_type", "keyword", "package_id", "package_name", "repo_name", "repo_owner", "score", "summary", "stars", "license", "last_commit_date", "last_activity_at" FROM "search", CONCAT("keywords") AS "keyword" WHERE CONCAT_WS(' ', "package_name", COALESCE("summary", ''), "repo_name", "repo_owner") ~* $1 AND "package_name" IS NOT NULL AND "repo_owner" IS NOT NULL AND "repo_name" IS NOT NULL ORDER BY LOWER("package_name") = $2 DESC, "score" DESC, "package_name" ASC"#)
+        _assertInlineSnapshot(matching: renderSQL(b), as: .lines, record: recordSQL, with: """
+        SELECT 'package' AS "match_type", "keyword", "package_id", "package_name", "repo_name", "repo_owner", "score", "summary", "stars", "license", "last_commit_date", "last_activity_at", "dependencies_count" FROM "search", CONCAT("keywords") AS "keyword", COALESCE(array_length("resolved_dependencies", 1), 0) AS "dependencies_count" WHERE CONCAT_WS(' ', "package_name", COALESCE("summary", ''), "repo_name", "repo_owner") ~* $1 AND "package_name" IS NOT NULL AND "repo_owner" IS NOT NULL AND "repo_name" IS NOT NULL ORDER BY LOWER("package_name") = $2 DESC, "score" DESC, "package_name" ASC
+        """)
```

Comment:
Using

Reply:
Actually, scratch that. `renderSQL` is a method on `AppTestCase` because it's using `app.db` to instantiate an `SQLSerializer`. We'd need to make it `.sql(app.db)` to pass it into the strategy and also make `renderSQL` and `bind` static or global functions. At that point it's just simpler to render it out into a `String` as is.